npm/webpack-dev-server/src/startServer.ts

```diff
@@ -48,13 +49,28 @@ export async function start ({ webpackConfig: userWebpackConfig, template, optio
   }
 
   debug('starting webpack dev server')
-
-  const webpackDevServerConfig: WebpackDevServer.Configuration = {
+  let webpackDevServerConfig: WebpackDevServer.Configuration = {
     ...userWebpackConfig.devServer,
     hot: false,
-    inline: false,
-    publicPath: devServerPublicPathRoute,
-    noInfo: false,
+  }
+
+  if (webpackDevServerPkg.version.match(/3\./)) {
+    webpackDevServerConfig = {
+      ...webpackDevServerConfig,
+      inline: false,
+      publicPath: devServerPublicPathRoute,
+      noInfo: false,
+    }
+  } else if (webpackDevServerPkg.version.match(/4\./)) {
+    webpackDevServerConfig = {
+      ...userWebpackConfig.devServer,
+      devMiddleware: {
```

Comment:
I needed to change this from devMiddleware to `dev` in the compiled js. 

Maybe because I'm running dwp 4 beta2?

Reply:
Yes. Just update to beta 3. See the [release notes](https://github.com/webpack/webpack-dev-server/releases/tag/v4.0.0-beta.3).

> the dev option was renamed to devMiddleware